package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "shutterstock",
+  "name": "shutterstock-api",
```

Comment:
how does this affect the generated code?

Reply:
This should not impact the generated code, especially because our generator assumes `shutterstock-api` already. https://github.com/shutterstock/public-api-sdk-generator/blob/master/build/sdk/js/templates/package.mustache#L2

Reply:
I believe it affects the generated documentation however. @timothymcmackin, thoughts?

Reply:
Not quite sure how this pipeline works. Shouldn't the package.json get generated from the mustache templates in public-api-sdk-generator and overwrite this one?